test/flint/fmpz_mat-test.jl

```diff
@@ -554,7 +554,13 @@ end
 
    r, N = nullspace(A)
 
-   @test iszero(A*N)
+    @test iszero(A*N)
```

Comment:
Just a minor thing, but: the rest of this file uses 3-space indentation, your new code has 4-space indentation. Anyway, from my POV that's not a blocker for merging, just wanted to mention it :-)

Reply:
Ah, yeah, thanks for pointing that out!